pkg/pullrequest/disk.go

```diff
@@ -169,6 +169,18 @@ func FromDisk(path string) (*Resource, error) {
 	var err error
 	var manifest Manifest
 
+	// If pr.json exists in output directory, preload r.PR
+	prPath := filepath.Join(path, "pr.json")
+	if _, err = os.Stat(prPath); err == nil {
+		b, err := ioutil.ReadFile(path + "/pr.json")
```

Comment:
ioutil.ReadFile(filepath.Join(path, "pr.json"))

Reply:
ha! or better still ioutil.ReadFile(prPath) .... changes incoming for updates.